surveys/sphinx-survey-2025.yml

```diff
@@ -268,7 +268,7 @@ questions:
   - "Sphinx Book"
   - "Sphinx Bootstrap"
   - "Sphinx Immaterial"
-  - "Something else/I don’t know"
+  - "I don’t know"
```

Comment:
"Something else" is already covered by the `other` option. - And it's important to distinguish "I don't know" from other answers.